src/Camelotia.Presentation/DesignTime/DesignTimeProviderViewModel.cs

```diff
@@ -63,6 +65,12 @@ public DesignTimeProviderViewModel()
 
         public string CurrentPath { get; } = "/home/files";
 
+        public IEnumerable<IFolderViewModel> BreadCrumbs { get; } = new List<IFolderViewModel>
+        {
+            new DesignTimeFolderViewModel {Name = "home", Children = new[] { new DesignTimeFolderViewModel { Name = "home" }, new DesignTimeFolderViewModel { Name = "home1" }, new DesignTimeFolderViewModel { Name = "home2" } }},
```

Comment:
Nit: worth splitting this line into multiple lines as there are folks who dislike horizontal scrolling.